oak_channel/SPEC.md

```diff
@@ -44,91 +43,47 @@ The service MUST NOT send request messages.
 ##### Handling an invocation
 
 The service MUST respond to each request message with a single response message.
-It SHOULD respond to request messages in first-in-first-out order, but MAY
-prioritize some request messages. The service MAY send a response message before
-the request message has been received in its entirety. Sending a response
-message concludes the invocation.
+The service MAY send a response message before the request message has been
+received in its entirety, but MUST NOT send start sending a response before any
```

Comment:
```suggestion
received in its entirety, but MUST NOT start sending a response before any
```

Reply:
How could a response message be sent before any part of the request has been received? How would the server know where to send the response to? Or is a connection already made at this stage?